lib/minesweeper_2pl/app.rb

```diff
@@ -1,7 +1,38 @@
 module Minesweeper_2pl
   class App
-    def call
-      "Hello World"
+    SIZE = 100
+    BOMB_COUNT = 10
+    attr_accessor :game, :cli
+
+    def start
+      self.setup
+      self.play_game
+    end
+
+    def setup
```

Comment:
The name of this suggests to me that it's similar to `initialize`, the method that Ruby will call first when you call `App.new`. How is `setup` different from `initialize`? If it's not, should this work just go in the `initialize` method of this object?